2021-10-28 17:48:08,296 INFO    MainThread:634996 [wandb_setup.py:_flush():69] setting env: {}
2021-10-28 17:48:08,296 INFO    MainThread:634996 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-28 17:48:08,296 INFO    MainThread:634996 [wandb_init.py:_log_setup():348] Logging user logs to /home/ingambe/DeepLearningCourse/3-Deep-networks/Slides/wandb/run-20211028_174808-ti0mdd55/logs/debug.log
2021-10-28 17:48:08,297 INFO    MainThread:634996 [wandb_init.py:_log_setup():349] Logging internal logs to /home/ingambe/DeepLearningCourse/3-Deep-networks/Slides/wandb/run-20211028_174808-ti0mdd55/logs/debug-internal.log
2021-10-28 17:48:08,298 INFO    MainThread:634996 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff10454c880>
2021-10-28 17:48:08,299 INFO    MainThread:634996 [wandb_init.py:init():381] calling init triggers
2021-10-28 17:48:08,299 INFO    MainThread:634996 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-10-28 17:48:08,300 INFO    MainThread:634996 [wandb_init.py:init():430] starting backend
2021-10-28 17:48:08,300 INFO    MainThread:634996 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-28 17:48:08,329 INFO    MainThread:634996 [backend.py:ensure_launched():135] starting backend process...
2021-10-28 17:48:08,348 INFO    MainThread:634996 [backend.py:ensure_launched():139] started backend process with pid: 635089
2021-10-28 17:48:08,352 INFO    MainThread:634996 [wandb_init.py:init():435] backend started and connected
2021-10-28 17:48:08,409 INFO    MainThread:634996 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-10-28 17:48:08,413 INFO    MainThread:634996 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-28 17:48:08,414 INFO    MainThread:634996 [wandb_init.py:init():494] updated telemetry
2021-10-28 17:48:08,416 INFO    MainThread:634996 [wandb_init.py:init():517] communicating current version
2021-10-28 17:48:13,419 INFO    MainThread:634996 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-28 17:48:14,626 INFO    MainThread:634996 [wandb_init.py:init():557] starting run threads in backend
2021-10-28 17:48:17,558 INFO    MainThread:634996 [wandb_run.py:_console_start():1605] atexit reg
2021-10-28 17:48:17,559 INFO    MainThread:634996 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-10-28 17:48:17,559 INFO    MainThread:634996 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-10-28 17:48:17,561 INFO    MainThread:634996 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-28 17:48:17,561 INFO    MainThread:634996 [wandb_init.py:init():582] run started, returning control to user process
2021-10-28 17:48:17,565 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:17,565 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:17,573 INFO    MainThread:634996 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:48:17,576 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:17,576 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:17,585 INFO    MainThread:634996 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:48:19,069 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:19,069 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:19,073 INFO    MainThread:634996 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:48:19,368 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:19,368 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:19,377 INFO    MainThread:634996 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:48:19,381 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:19,381 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:19,389 INFO    MainThread:634996 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:48:21,347 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:21,347 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:21,351 INFO    MainThread:634996 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:48:21,661 INFO    MainThread:634996 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:48:21,661 INFO    MainThread:634996 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:48:53,800 INFO    MainThread:634996 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-10-28 17:48:53,801 INFO    MainThread:634996 [wandb_run.py:_restore():1547] restore
2021-10-28 17:48:54,147 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 810
}

2021-10-28 17:48:54,780 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 810
}

2021-10-28 17:48:55,720 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  total_bytes: 10314
}

2021-10-28 17:48:55,827 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10540
}

2021-10-28 17:48:55,931 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10540
}

2021-10-28 17:48:56,035 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10540
}

2021-10-28 17:48:56,139 INFO    MainThread:634996 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10540
}

